src/format.c

```diff
@@ -356,8 +396,19 @@ void write_csp(char dump_type, char *filename) {
   int32_t i, j;
```

Comment:
Declare loop variables in loops, remove from top of function.

Reply:
This is some original ReSCAL code, I just indented it. Declarations moved to loops.

Reply:
Thanks.